indico/modules/events/abstracts/templates/display/user_abstract_list.html

```diff
@@ -0,0 +1,65 @@
+{% from 'events/display/indico/_common.html' import render_users %}
+
+{% extends 'layout/conference_page_base.html' %}
+
+{% macro render_abstract_list(event, abstracts) %}
+    {% set mapping = {'under_review': 'highlight',
+                      'withdrawn': 'outline dashed',
+                      'accepted': 'success',
+                      'rejected': 'error',
+                      'merged': 'visited',
+                      'duplicate': 'strong'} %}
+    {% for abstract in abstracts -%}
+        <div class="contribution-row" data-friendly-id="{{ abstract.friendly_id }}">
+            <div class="contrib-title info">
+                <span class="value">
+                    <a class="js-mathjax" href="{{ url_for('abstracts.display_abstract', abstract) }}">
+                        <span class="contrib-id">{{ abstract.friendly_id }}.</span>
+                        {{- abstract.title -}}
+                    </a>
+                </span>
+            </div>
+            <div class="speaker-list icon-user">
+                {{ render_users(abstract.speakers|sort(attribute='display_order_key'),
+                                span_class='speaker-item-inline') }}
+            </div>
+            <div class="contrib-time icon-time">
+                {% set dt = abstract.modified_dt or abstract.submitted_dt %}
+                <span>{% trans %}Last modified:{% endtrans %}</span>
+                <span>{{ dt|format_datetime(format='short', timezone=timezone) }}</span>
+            </div>
+            <div>
+                <div class="contrib-state small i-tag {{ mapping[abstract.public_state.name] }}">
+                    {{- abstract.state.title -}}
+                </div>
+                {% if abstract.accepted_track -%}
+                    <div class="contrib-track small">
+                        {{- abstract.accepted_track.title -}}
+                    </div>
+                {%- endif %}
+                {% if abstract.accepted_contrib_type -%}
+                    <div class="contrib-type small">
+                        {{- abstract.accepted_contrib_type.name -}}
+                    </div>
+                {%- endif %}
+            </div>
+            {% if abstract.description -%}
+                <a href="{{ url_for('abstracts.display_abstract', abstract) }}">
+                    <div class="description js-mathjax">
+                        {{- abstract.description|truncate(400) -}}
+                    </div>
+                </a>
+            {%- endif %}
+        </div>
+    {%- endfor %}
+{% endmacro %}
+
+{% block title -%}
+    {% trans %}My Abstracts{% endtrans %}
```

Comment:
haha, I just think "My Abstracts" sounds better. It's quite obvious that those are the ones you submitted